src/OpenLoco/src/Ui/WindowManager.cpp

```diff
@@ -556,11 +556,11 @@ namespace OpenLoco::Ui::WindowManager
         // Border flash invalidation
         for (Ui::Window* w = _windowsEnd - 1; w >= _windows; w--)
         {
-            if ((w->flags & WindowFlags::whiteBorderMask) != 0)
+            if (w->hasFlags(WindowFlags::whiteBorderMask))
             {
                 // TODO: Replace with countdown
-                w->flags -= WindowFlags::whiteBorderOne;
-                if ((w->flags & WindowFlags::whiteBorderMask) == 0)
+                w->flags &= ~WindowFlags::whiteBorderOne;
```

Comment:
Argh I didn't read this code it was meant to do a -=. Can you change it to the following:
```cpp
if (w->hasFlags(WindowFlags::whiteBorderMask)){

    // TODO: Replace with seperate variable countdown
    // Countdown is stuffed into WindowFlags extract it out and decrement by 1
    const auto newCount = enumValue(w->flags & WindowFlags::whiteBorderMask) - enumValue(WindowFlags::whiteBorderOne);
    // Stuff the new count back into the WindowFlags
    w->flags &= ~WindowFlags::whiteBorderMask;
    w->flags |= static_cast<WindowFlags>(newCount);

    if (!w->hasFlags(WindowFlags::whiteBorderMask)){

Reply:
I see, we should maybe only allow windows flags access by getter/setter for as long the count is also encoded in there, should also make this a bit more readable again. Not required for this PR